### PR TITLE
New documentation entry for registration invites path.

### PR DESCRIPTION
**Pull request by clrobinson-traction**
### Description
With this PR being merged/deployed (https://github.com/tractionguest/guest-server/pull/1533), a new endpoint needed to be documented. Changes should be reasonably clear, since it's mostly just one block of additions.

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -1808,6 +1808,122 @@ paths:
...
+                    mobile_number: some text
+                    invite_watchlist:
+                      id: 3
+                      external_colours:
+                        - some text
+                      internal_colours:
+                        - some text
+                        - some text
+          description: Created an `Invite` from the given `Registration` UUID.
+        '400':
```
Comment (brandonjmckay) [inline openapi.yml:1892]: @stevenallen05 were you doing some weird `4xx` thing for these now?
I remember it seemed that Shabini just had to do a fix for her added paths not including responses for the other 4xx responses other than 400.

**Comment by brandonjmckay** [comment]
No wrike ticket?

File: openapi.yml (hunk 1 above)
Comment (stevenallen05) [reply openapi.yml:1892]: It's not mandatory to do it either way. It just saves on the copy/paste to do the `4XX` wildcard.

**Comment by clrobinson-traction** [comment]
No, no ticket. I suppose we should've made one. I just remembered to open this PR after completing the dev ticket.

We **really** need to formalize/document how to work with API v3, from a dev team standpoint.

**Pull request merged**